README.md

```diff
@@ -75,10 +75,17 @@ This change in namespace to `node['postfix']['main']` should allow for greater f
 - `node['postfix']['main']['smtp_sasl_password_maps']` - Set to `hash:/etc/postfix/sasl_passwd` template file
 - `node['postfix']['main']['smtp_sasl_security_options']` - Set to noanonymous
 - `node['postfix']['main']['relayhost']` - Set to empty string
-- `node['postfix']['sasl']['smtp_sasl_user_name']` - SASL user to authenticate as. Default empty
-- `node['postfix']['sasl']['smtp_sasl_passwd']` - SASL password to use. Default empty.
 - `node['postfix']['sender_canonical_map_entries']` - (hash with key value pairs); default not configured. Setup generic canonical maps. See `man 5 canonical`. If has at least one value, then will be enabled in config.
 - `node['postfix']['smtp_generic_map_entries']` - (hash with key value pairs); default not configured. Setup generic postfix maps. See `man 5 generic`. If has at least one value, then will be enabled in config.
+- `node['postfix']['sasl']` = ```json {
```

Comment:
I'm thinking maybe leave the old attributes in and note that they are only in use up to current version of cookbook, and then note these attributes as only available since the version that will be coming. Since we do the merging and the versioning you don't need to put the version info in there but this is a note/reminder to any of the folks that are looking at this to make sure this is noted.